### PR TITLE
Separate tests

### PR DESCRIPTION
**Pull request by crookedneighbor**
**Description**

Pulls the various test classes into their own files.

**Checklist**

- [ ] ~~Add tests for these changes~~
- [x] Test coverage is not decreased
- [x] Entire test suite passes

### CODE REVIEW

**Comment by crookedneighbor** [comment]
I don't write in python very often, so I'm sure there's some stuff I'm doing here that is not very pythonic. As always, feel free to change anything.

File: tests/test_zmeta.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from warnings import warn
+
+from helpers.constants import REPO_ROOT, S_SPY, SNAPSHOTS_USED  # type:ignore
+
+from spellbot.assets import load_strings
+
+
+# These tests will fail in isolation, you must run the full test suite for them to pass.
+# The file has a leading z in the test name to ensure it gets run last
```
Comment (crookedneighbor) [inline tests/test_zmeta.py:9]: Not sure if there's a good alternative than to just make this file have a `z` before the test name

Comment (lexicalunit) [reply tests/test_zmeta.py:9]: I may try and add https://pytest-ordering.readthedocs.io/en/develop/ to do this if you haven't already by the time I get to it!

Comment (crookedneighbor) [reply tests/test_zmeta.py:9]: Perfect, this worked great.

**Comment by lexicalunit** [comment]
I wasn't sure how to update a PR, so I pushed up some changes to a new PR: https://github.com/lexicalunit/spellbot/pull/137 -- Once it passes tests I'll merged it. Thanks for the awesome work!

**Comment by crookedneighbor** [comment]
:+1:

**Pull request closed without merging**